server/test/repositories/metadata.repository.mock.ts

```diff
@@ -4,7 +4,9 @@ export const newMetadataRepositoryMock = (): jest.Mocked<IMetadataRepository> =>
   return {
     deleteCache: jest.fn(),
     getExifTags: jest.fn(),
-    init: jest.fn(),
+    initLocalGeocoding: jest.fn(),
+    deinitMapboxGeocoding: jest.fn(),
+    initMapboxGeocoding: jest.fn(),
```

Comment:
Instead of various, provider specific methods, we could have a single one that takes in generic options.

I think lookup can take in a point, and then options which specify the provider type and optionally the access key. Basically use a single method for all providers and abstract the options.